python/servo/package_commands.py

```diff
@@ -209,24 +209,21 @@ def package(self, release=False, dev=False, android=None, debug=False, debugger=
             msi_path = path.join(dir_to_msi, "Servo.msi")
             print("Packaged Servo into {}".format(msi_path))
         else:
+            dir_to_temp = '/'.join(binary_path.split('/')[:-2]) + '/targz'
```

Comment:
It's the Python slice operator. In a Python2 REPL:
```
[aneesh@cheetah servo]$ python2                                                                       [18:59:10]
Python 2.7.12 (default, Jun 28 2016, 08:31:05)
[GCC 6.1.1 20160602] on linux2
Type help, copyright, credits or license for more information.
>>> binary_path = '/home/aneesh/workspace/servo'
>>> binary_path.split('/')
['', 'home', 'aneesh', 'workspace', 'servo']
>>> binary_path.split('/')[:-2]
['', 'home', 'aneesh']
>>> [only_one_element_here][:-2]
[]
```

Essentially, it takes a list and returns another list with the last two elements removed. Here it's being used to strip off two path components from `binary_path`.